### ISSUE TITLE
create-react-app - Failed to minify the code from this file

### ISSUE THREAD

**Issue by cezarlz** [gtournie/redux-form-validators#28]
When I try to build my production build I got:

Versions tested: 2.4.1, 2.3.2

2.2.0 works.

```js
> react-scripts build

Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file:

 	./node_modules/redux-form-validators/src/helpers.js:20

Read more here: http://bit.ly/2tRViJ9
```

**Comment by ChrisSargent** [comment]
Just having the same issue. Seems to be because the package.json file has:

"module": "src/index.js",

changing to

"module": "lib/index.js", works.

**Comment by cezarlz** [comment]
Exactly :)

**Comment by dszymczuk** [comment]
I have same issue.

**Comment by gtournie** [comment]
Fixed in 2.4.2
